charts/azure-k8s-metrics-adapter/templates/cluster-role.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "azure-k8s-metrics-adapter.fullname" . }}
+  labels:
+    app: {{ template "azure-k8s-metrics-adapter.name" . }}
+    chart: {{ template "azure-k8s-metrics-adapter.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
```

Comment:
Should be configurable.  These give access to which components it can look information up for.

Reply:
Got it, I'll use this as the default.